### PR TITLE
Deal better with ties for maximum frequency

### LINKED ISSUE #974: Nightly build broken (due to a stochastic test?)
Of course, the first time the nightly build fails is after release 🤦 

I am not sure why a test that [passed in a PR](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/968) is now failing on [nightly](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r).

My best guess that there is some stochastic element here that makes `.createAxesLabels()` produce the `yLabel` to be sometimes `"Concentration [µmol/l]"`, while other times `"Concentration [mg/l]"` in the context of this specific test:

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000
options(tibble.width = Inf)

concDataSet <- DataSet$new(name = "Concentration data set")
concDataSet$setValues(1, 1)
concDataSet$yDimension <- ospDimensions$`Concentration (molar)`
concDataSet$molWeight <- 1

amountDataSet <- DataSet$new(name = "Amount data set")
amountDataSet$setValues(1, 1)
amountDataSet$yDimension <- ospDimensions$`Concentration (mass)`
amountDataSet$molWeight <- 1

myCombDat <- DataCombined$new()
myCombDat$addDataSets(c(concDataSet, amountDataSet))

(df <- myCombDat$toDataFrame())
#> # A tibble: 2 x 14
#>   name                   group dataType xValues xUnit xDimension yValues yUnit 
#>   <chr>                  <chr> <chr>      <dbl> <chr> <chr>        <dbl> <chr> 
#> 1 Amount data set        <NA>  observed       1 h     Time          1.00 mg/l  
#> 2 Concentration data set <NA>  observed       1 h     Time          1    µmol/l
#>   yDimension            yErrorValues yErrorType yErrorUnit molWeight  lloq
#>   <chr>                        <dbl>      <dbl>      <dbl>     <dbl> <dbl>
#> 1 Concentration (mass)            NA         NA         NA         1    NA
#> 2 Concentration (molar)           NA         NA         NA         1    NA

(df <- ospsuite:::.unitConverter(df))
#> # A tibble: 2 x 14
#>   name                   group dataType xValues xUnit xDimension yValues yUnit 
#>   <chr>                  <chr> <chr>      <dbl> <chr> <chr>        <dbl> <chr> 
#> 1 Amount data set        <NA>  observed       1 h     Time         1000. µmol/l
#> 2 Concentration data set <NA>  observed       1 h     Time            1  µmol/l
#>   yDimension            yErrorValues yErrorType yErrorUnit molWeight  lloq
#>   <chr>                        <dbl>      <dbl> <chr>          <dbl> <dbl>
#> 1 Concentration (mass)            NA         NA µmol/l             1    NA
#> 2 Concentration (molar)           NA         NA µmol/l             1    NA

ospsuite:::.createAxesLabels(df, tlf::TimeProfilePlotConfiguration$new())
#> $xLabel
#> [1] "Time [h]"
#> 
#> $yLabel
#> [1] "Concentration [µmol/l]"
```

<sup>Created on 2022-05-23 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #974

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
Took 12 minutes 👀

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite
 Title: R package to manipulate OSPSuite Models
-Version: 11.0.0
+Version: 12.0.0
```
Comment (Yuri05) [inline DESCRIPTION:4]: Now we should set the version to 12.0.0.**9000** , right? In order to avoid the production docs are overwritten by dev docs

Comment (IndrajeetPatil) [reply DESCRIPTION:4]: Sure. Done so in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/977.

Comment (IndrajeetPatil) [reply DESCRIPTION:4]: Are nightly builds failing because we bumped the version to `12.x.x` here and AppVeyor is expecting `11.x.x`?

**Pull request merged**